backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/progressbar/BpkProgressBar.kt

```diff
@@ -0,0 +1,114 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.compose.progressbar
+
+import androidx.compose.animation.core.animateFloatAsState
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.LinearProgressIndicator
+import androidx.compose.material3.ProgressIndicatorDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.StrokeCap
+import androidx.compose.ui.unit.dp
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import androidx.compose.ui.unit.Dp
+
+enum class BpkProgressBarSize {
+    Small,
+    Large,
+}
+
+@Composable
+fun BpkProgressBar(
+    value: Float,
+    // accessibilityLabel: String, <- not sure if we need this, in its current state talkback reads ("zero percent progress bar")
+    modifier: Modifier = Modifier,
+    size: BpkProgressBarSize = BpkProgressBarSize.Small,
+    min: Int = 0,
+    max: Int = 1,
+    stepped: Boolean = false,
+) {
+    val height = if (size == BpkProgressBarSize.Small) 8.dp else 16.dp
```

Comment:
can you use the `BpkSpacing` values please?

Reply:
Done